### PR TITLE
update DOLFIN easyblock for v1.6.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyblocks/d/dolfin.py [hunk 1]
```diff
@@ -121,8 +135,14 @@ def configure_step(self):
         self.boost_dir = depsdict['Boost']
 
         # UFC and Armadillo config params
-        self.cfg.update('configopts', " -DUFC_DIR=%s" % depsdict['UFC'])
-        self.cfg.update('configopts', "-DARMADILLO_DIR:PATH=%s " % depsdict['Armadillo'])
+        if 'UFC' in depsdict:
+            self.cfg.update('configopts', " -DUFC_DIR=%s" % depsdict['UFC'])
```
Comment (wpoely86) [inline easybuild/easyblocks/d/dolfin.py]: whitespace before or after, be consistent.

File: easybuild/easyblocks/d/dolfin.py [hunk 2]
```diff
@@ -178,6 +201,107 @@ def configure_step(self):
...
+            for libdir in ['lib', 'lib64']:
+                if os.path.exists(os.path.join(self.installdir, libdir, dolfin_lib)):
+                    dolfin_libdir = os.path.join(self.installdir, libdir)
+                    break
+            if dolfin_libdir is None:
+                raise EasyBuildError("Failed to locate %s", dolfin_lib)
+
+            for pylibdir in self.all_pylibdirs:
+                libs = glob.glob(os.path.join(self.installdir, pylibdir, 'dolfin', 'cpp', '_*.so'))
+                for lib in libs:
```
Comment (wpoely86) [inline easybuild/easyblocks/d/dolfin.py]: add debuging print about what you're gonna do

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Tested with easyconfig for DOLFIN 1.6 from https://github.com/hpcugent/easybuild-easyconfigs/pull/2271.

Going in, thanks for the review @wpoely86!

**Pull request merged**